src/Lykke.AlgoStore.CSharp.AlgoTemplate.Models/Repositories/AlgoInstanceTradeRepository.cs

```diff
@@ -115,7 +116,7 @@ public async Task<IEnumerable<AlgoInstanceTrade>> GetInstaceTradesByTradedAssetA
 
             var result = new List<AlgoInstanceTrade>();
 
-            await _tableStorage.GetDataByChunksAsync(query, (items) => result.AddRange(AutoMapper.Mapper.Map<List<AlgoInstanceTrade>>(items)));
+            await _tableStorage.GetDataByChunksAsync(query, (items) => result.AddRange(AutoMapper.Mapper.Map<List<AlgoInstanceTrade>>(items))).ContinueWith(t => { }, ct);
```

Comment:
This won't work the way you expect it to - ContinueWith starts only after the previous task has finished